spot-contracts/contracts/BondIssuer.sol

```diff
@@ -50,10 +56,15 @@ contract BondIssuer is IBondIssuer, OwnableUpgradeable {
     ///      then the issue window opens at Friday 2AM GMT every week.
     uint256 public issueWindowOffsetSec;
 
-    /// @notice An enumerable list to keep track of bonds issued by this issuer.
-    /// @dev Bonds are only added and never removed, thus the last item will always point
-    ///      to the latest bond.
-    EnumerableSetUpgradeable.AddressSet private _issuedBonds;
+    /// @dev An list of all active bonds, i.e) bonds which have NOT mature.
+    EnumerableSetUpgradeable.AddressSet private _activeBonds;
+
+    /// @dev An list of "inactive" or mature bonds.
+    ///      Once a bond matures its moved from the `active` list into this one.
+    EnumerableSetUpgradeable.AddressSet private _matureBonds;
```

Comment:
Apologies this might seem really minor, but we could rename this to `_maturedBonds`.

This makes it clear that the bonds here have explicitly "been matured", meaning that mature() has been executed.
I make a distinction between this state and the state where a bond has reached the maturity time, but hasn't yet been matured in contract storage.

Reply:
Yea that makes sense 👍